spyder/config/main.py

```diff
@@ -500,6 +500,11 @@
               'advanced/external': False,
               'advanced/stdio': False
              }),
+            ('kite',
+             {
+              'enable': True,
+              'enable_snippets': True,

```

Comment:
@metalogical, how does an user can disable code snippets for Kite?

Reply:
currently they cannot (apart from editing the config file), but as a follow-up, we should expose this setting in a UI

Reply:
Sounds reasonable

Reply:
I will update the UI after this PR is merged!

Reply:
@andfoy @ccordoba12 since this adds even more granularity. Since we will use one or the other, we do not need to expose a setting for Kite AND a setting for the PYLS right?

Reply:
Personally, I would prefer to keep a single option to set all the completion configuration options. However, this will fixed as part of a future completion refectoring that we plan to do for Spyder 5. Maybe we should stick to a single configuration value, or set this one using the current checkbox, otherwise users would get completions even if they explicitly set that option disabled.

Reply:
ok, I'm fine with keeping a single config option. I just thought it was strange that we had to check the LSP config option inside of the Kite plugin. Will revert to the previous way.

Reply:
Yes, it's a little bit confusing to use LSP references everywhere, but we will do the corresponding refactoring later on